axoproject/src/rust.rs

```diff
@@ -200,6 +198,19 @@ fn package_info(_workspace_root: &Utf8Path, package: &PackageMetadata) -> Result
             )
         };
 
+    let axoupdater_version = if package.name() == "axoupdater" {
+        Some(Version::Cargo(package.version().to_owned()))
+    } else {
+        let mut version = None;
+        for subpackage in package.direct_links() {
```

Comment:
what would it mean if someone transitively depended on axoupdater (trying to sort if holding the transitive user of axoupdater to the dist version restrictions makes sense)

Reply:
at the very least, i think we should preserve the information that it's depended on transitively as that will be very useful for the end user receiving the error message